.github/workflows/kind-e2e.yaml

```diff
@@ -166,6 +169,32 @@ jobs:
 
         kind create cluster --config kind.yaml
 
+    - name: Configure NetworkPolicy
+      working-directory: ./src/knative.dev/serving
+      run: |
+        set -x
+
+        cat >> netpolicy.yaml <<EOF
+        apiVersion: networking.k8s.io/v1
+        kind: NetworkPolicy
+        metadata:
+          name: default-deny
+          namespace: knative-serving
+        spec:
+          podSelector:
+            matchExpressions:
+            - key: app
+              operator: In
+              values: [autoscaler, autoscaler-hpa]
+          policyTypes:
+          - Ingress
+        EOF
+
+    - name: Setup NetworkPolicy CNI
+      run: |
+        sleep 10
```

Comment:
I don't fully follow why we need the sleep here (I could somewhat understand needing it _after_ CNI was installed, but why before?). It doesn't look like the `Install Serving & Ingress` step needs a sleep at the start, do we understand why this step does?